src/Pages/SinglePageVideo/SinglePageVideo.jsx

```diff
@@ -1,15 +1,40 @@
 import { useParams } from "react-router-dom";
-import { BiLike, BiDislike } from "react-icons/bi";
+import { HiThumbUp, HiOutlineThumbUp } from "react-icons/hi";
```

Comment:
you can export all icons from one index file, and import any of the icons from that.